src/frontend/utils/formatter.ts

```diff
@@ -18,6 +20,7 @@ export const formaterIsoDato = (
     tilFormat: datoformat,
     defaultString?: string
 ) => {
+    moment.locale('nb');
```

Comment:
Jeg tror dette setter locale til moment for alle instanser den blir brukt siden det er en klasse. Jeg ville vurdert å legge det ut til en egen fil hvor man oppretter moment én gang og henter den derfra overalt hvis vi skal sette locale.

Reply:
Ja, denne skulle vi flytte ut, men gikk i glemmeboka. Hvis vi uansett har norsk som språk funker det kanskje å sette dette allerede i index-fila?

Reply:
Jepp, det vil jeg tro! 👍